include/verilatedos.h

```diff
@@ -234,7 +234,7 @@
 //=========================================================================
 // C++-2011
 
-#if __cplusplus >= 201103L || defined(__GXX_EXPERIMENTAL_CXX0X__) || defined(VL_CPPCHECK)
+#if __cplusplus >= 201103L || defined(__GXX_EXPERIMENTAL_CXX0X__) || defined(VL_CPPCHECK) || defined(_MSC_VER)
 #else
```

Comment:
Please put into a "fix MSVC" first pull.

I think you need to check for a recent enough MSC_VER. https://stackoverflow.com/questions/5047971/how-do-i-check-for-c11-support